packages/terra-application/src/application-loading-overlay/ApplicationLoadingOverlay.jsx

```diff
@@ -0,0 +1,41 @@
+import { useRef, useContext, useLayoutEffect } from 'react';
+import PropTypes from 'prop-types';
+import uuidv4 from 'uuid/v4';
+
+import ApplicationLoadingOverlayContext from './ApplicationLoadingOverlayContext';
+
+const propTypes = {
+  /**
+   * A boolean value indicating whether the loading overlay should be visible or not.
+   */
+  isOpen: PropTypes.bool,
+  /**
+   * A string to be rendered within the loading overlay when open.
+   */
+  message: PropTypes.string,
```

Comment:
The say the loading overlay provider is implemented, it seems like this prop is required because the Loading Overlay's default message doesn't seem like it would be hit. Should this be required?

Reply:
Hmm, that's just a bug. If no messages are provided, we should default back to the default string.

Reply:
@tbiethman were changes made to fix this??

Reply:
The messages were removed entirely due to future UX recommendations.